### PR TITLE
Fix nitpicks

### PR DESCRIPTION
**Pull request by bojanz**
I've been reading the Rules code tonight, came across some silly issues, decided to fix them.

Remarks:
- The config entities should have matching interfaces
- It's weird for entities to define links provided by the rules_ui module. Maybe it's time to kill the UI separation?

### CODE REVIEW

**Comment by fubhy** [comment]
> It's weird for entities to define links provided by the rules_ui module. Maybe it's time to kill the UI separation?

Absolutely. We spoke about that a while ago and IIRC there was no argument for keeping the UI module separate apart from other priorities atm. ... But I am all for it.

File: src/Entity/ReactionRule.php [hunk 1]
```diff
@@ -135,9 +135,6 @@ public function getExpression() {
   /**
    * Returns the Rules expression manager.
    *
-   * @todo Actually we should use dependency injection here, but is that even
-   *   possible with config entities? How?
```
Comment (klausi) [inline src/Entity/ReactionRule.php:139]: Hm, should we remove that todo? We are using \Drupal here, which we should not do, so we should document why we do it.

**Comment by klausi** [comment]
Merged everything except the todos, thanks!

**Comment by bojanz** [comment]
Well, there's no way not to do it, D8 doesn't support DI in entities, typed data, a few other places. Rules is doing the same thing core is doing. Anyway, thanks for the merge.

**Pull request closed without merging**